packages/svelte/tests/compiler-errors/samples/css-global-block-declaration/main.svelte

```diff
@@ -1,5 +1,19 @@
 <style>
+	/* ok */
 	.x :global {
 		color: red;
 	}
+
+	:global .y {
+		color: red;
+	}
+
+	:global.y {
+		color: red;
+	}
```

Comment:
it feels like this should be disallowed — `:global.y` is equivalent to this...

```css
:global {
  &.y {...}
}
```

...which doesn't make any sense, since the `&` doesn't _mean_ anything in this context. (We would also need to error on `&` inside standalone `:global {...}` blocks, which we probably should have been doing already.)

Reply:
If we disallow this then this won't fix the tailwind case. Tailwind turns this
```css
main :global { @apply foo dark:bar }
```
into
```css
main :global { ... }
main :global:where(dark-mode *) { ... }
```

Reply:
Yeah, that's exactly what I mean